src/app/components/ReadonlyEditorInterface.tsx

```diff
@@ -0,0 +1,461 @@
+'use client'
```

Comment:
do we need this component? i think this is not used anywhere. Also if we are deciding to recreate another editor instance just for the readonly mode i dont think this would be the best approach.

Reply:
@elhamzaouiaymen This was only an approach while trying out to solve autofill fields issue. I will take this file out in the next commit.